docs/hsi-tests.d/org.fwupd.hsi.Smap.json

```diff
@@ -1,6 +1,6 @@
 {
-  "id": "org.fwupd.hsi.IntelSmap",
-  "name": "Intel SMAP",
+  "id": "org.fwupd.hsi.Smap",
```

Comment:
same -- old ID needed here too

Reply:
Ohh, I just worked out this is the wrong PR -- can you fix up the other one please, and then I'l rebase this one on top --- thanks!

Reply:
fixed